src/_articles/archive/zones.md

```diff
@@ -152,33 +143,47 @@ Code might execute in other nested child zones as well,
 but at a minimum it always runs in the root zone.
 
 
-## Handling asynchronous errors
+## Handling uncaught errors
 
 One of the most used features of zones is
-their ability to handle uncaught errors
-in asynchronously executing code.
-The concept is similar to a try-catch in synchronous code.
-
-An _uncaught error_ is often caused by some code using `throw`
-to raise an exception that is not handled by a `catch` statement.
-Another way to produce an uncaught error is
-to call `new Future.error()` or
-a Completer's `completeError()` method.
-
-Use the `onError` argument to `runZoned()` to
-install a _zoned error handler_—an asynchronous error handler
-that's invoked for every uncaught error in the zone.
-For example:
+their ability to catch and handle uncaught errors.
+
+_Uncaught errors_ often occur because of code using `throw`
+to raise an exception without an accompanying `catch`
+statement to handle it.
+Uncaught errors can also arise in `async` functions
+when a Future completes with an error result,
+but is missing a corresponding `await` to handle the error.
+
+An uncaught error always reports to the current zone which 
+failed to catch it. Zones' default behavior is to handle uncaught
+errors by crashing the program.
+Instead of letting your program terminate,
+you can install your own custom _uncaught error handler_ to a new zone
+to intercept and handle uncaught errors however you prefer.
+
+The simplest way to introduce a new zone with an uncaught error handler
+is to use `runZoneGuarded`. Its `onError` callback becomes the
+uncaught error handler of a new zone, and handles any
+synchronous errors thrown by the call as well.
```

Comment:
**issue:** This could be simplified.
```suggestion
To introduce a new zone with an uncaught error handler,
use the `runZoneGuarded` method. Its `onError` callback becomes the
uncaught error handler of a new zone. This callback handles any
synchronous errors that the call throws.
```